cmake/USDPluginTools.cmake

```diff
@@ -295,11 +304,24 @@ function(usd_python_test TEST_TARGET PYTHON_FILE)
 
     # Set-up runtime environment variables for the test.
     # The paths refer to the build tree (which mirrors the final installation).
+    set(TEST_PYTHON_PATH 
+        "PYTHONPATH=${PROJECT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}/python" 
+        "${USD_ROOT}/${CMAKE_INSTALL_LIBDIR}/python" 
+        "$ENV{PYTHONPATH}"
+        "\\;PXR_PLUGINPATH_NAME=${_TEST_PXR_PLUGIN_PATH}")  # For Linux we need the semicolon in front
```

Comment:
Ohh I haven't seen the two backslashes before.  Is that required? (Could you explain what it is for?)

Reply:
Sure! So last night I bumbled through getting the Linux tests to pass a few times (sorry if there were a few extra emails!), and the double-escape sequence is here because of the output in:

This functioning line:
https://github.com/wetadigital/USDPluginExamples/pull/23/checks?check_run_id=1811841093#step:7:18

And this non-functioning line with triple colons:
https://github.com/wetadigital/USDPluginExamples/runs/1811709239?check_suite_focus=true#step:7:18

I'm a little fuzzy on the specifics, but my working theory is that CMake treats `;` as a list separator when lexing the code, and you first need to escape that functionality with a '\', and then you need to escape the semicolon character with another '\'. This seems to match random SO searches and the language reference here:

https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#escape-sequences

On MSVC, the common wisdom seems to be that you actually HAVE to double escape a semicolon in a string which is why I set it as the path separator, and my code wouldn't build when I didn't do it basically.

I did not test a single-escape here admittedly, but this seemed to be the safest choice for Linux and Windows 😄

Reply:
Thanks for the research & explanation, that makes sense! :-)